app/models/miq_worker.rb

```diff
@@ -455,6 +455,17 @@ def clean_active_messages
     end
   end
 
+  def error_out_tasks_with_active_queue_message
+    message = "Task Handler: [#{friendly_name}] ID [#{id}] has been deleted!"
+    processed_messages.each do |m|
+      task = m.miq_task
+      next unless task
+
+      args = [MiqTask::STATE_FINISHED, MiqTask::STATUS_ERROR, message]
+      MiqQueue.put(:class_name => task.class.name, :instance_id => m.miq_task_id, :method_name => "update_status", :args => args)
```

Comment:
Think this might read better as

```suggestion
      MiqQueue.put(
        :class_name  => task.class.name,
        :instance_id => m.miq_task_id,
        :method_name => "update_status",
        :args        => [MiqTask::STATE_FINISHED, MiqTask::STATUS_ERROR, message]
      )
```

Reply:
Yeah I'm not sure it's worth it to queue this up...I guess it depends on how many statuses you think there will be?

Reply:
I removed the queueing. Can put it back if needed later.  It's fast so it felt like async would be overkill here.